compiler/testData/codegen/composeLikeBytecodeText/default.kt

```diff
@@ -0,0 +1,8 @@
+fun foo(s: String = "O") = s
+
+fun box() = foo() + foo("K")
+
+// For Compose special default arugment handling, we still do not want
+// the default argument mask in the local variable table.
+
+// 0 LOCALVARIABLE \$default
```

Comment:
Maybe just `0 \$default` to check that there are no `$default` methods at all? Local variable names aren't a part of ABI, so they can possibly change in the future and the test will not check what it's checking now anymore

Reply:
Done.